.github/workflows/tests.yml

```diff
@@ -436,10 +436,11 @@ jobs:
       - name: Reclaim some memory
         uses: ./.github/actions/reclaim-memory
 
-      - name: Remove lxd-installer
+      - name: Purge LXD snap
+        # TODO: drop this when moving away from `ubuntu-22.04` runners to `ubuntu-24.04`
```

Comment:
We don't need to purge the snap as this part runs on 24.04

Reply:
Right, I fixed the commit message and went a bit further to stop doing this bit of the cleanup because for TiCS, LXD is never executed so the snap or the `lxd-installer` presence shouldn't matter, right?